### DISCUSSION TITLE
contributing in GSOC Open Source Project(2023)

### DISCUSSION THREAD

**Discussion by rajeck1234** [carbon-language/carbon-lang#2669] [General]
Hlw
I am Utkarsh Kumar FROM IIT KHARAGPUR 3RD YEAR Undergraduate student(BTECH).

The project I am interested in :

Improve the user-facing Carbon Explorer output
Availability 

Ø I have good experience with Java, C, C++, Python and basic Golang and JavaScript and good with programming skills. I also have a basic knowledge of Android and web development.

 

Ø I also did an onsite internship at  Rannlab Technology where I made project on Employee Management System using Python and SQl.

 

Ø I am also a sport programmer. I am a 4 star Hackerrank in C++. I am a consistent Leetcoders and solve 2 questions of Leetcode everyday.

 

 

Ø  I have made 3-4 projects. Some of them are an  Chabot, Web scrapping and sorting algorithm  using python.

 

Ø  Also have basic knowledge of open source contribution and earlier made few contribution on GitHub.

 

Ø  I am also a RESEARCH GROUP OF OUR COLLEGE.

 

I am available for full time work in this summer as I have no vacation plan other than this project. In the official GSoC period, I will devote the sufficient time per week which is required for this project. It’s my honour to be the part of this event. I will remain in contact in all time through the [e-mail](mailto:kumarutkarsh946@gmail.com) and on any other communication channel with the community

Other Commitments

Do you have any other commitments during the GSoC period?
 

           No, I am full time available to work with [Google Summer of Code](https://summerofcode.withgoogle.com/).

  

Have you applied to any other organizations? If so, to whom and do you have a preferred project/org?
 

No. I am totally focused on this event.

 

How you will be able to complete the project?
 

           I will devote the sufficient time per week which is required for this   

           Project and will complete the project within the given timeline.

 

How to approach you during Gsoc?
 

          I will remain in contact in all time through the [e-mail](mailto:kumarutkarsh946@gmail.com) and on any   

          other communication channel with the community.

Contact Info

 

UNIVERSITY

UNIVERSITY NAME: IIT KHARAGPUR

CURRENT YEAR: 3rd year

EXPECTED GRADUATION YEAR: 2024

DEGREE: B.Tech

 

Personal

EMAIL: [kumarutkarsh946@gmail.com](mailto:kumarutkarsh946@gmail.com)

 

INSTANT MESSAGING: Telegram or Whatsapp - 6204808269

**Comment by jonmeow** [comment]
Please note that per [GSoC's timelines](https://developers.google.com/open-source/gsoc/timeline), applications are not currently being accepted. When applications open up on March 20, I think you'll need to apply through GSoC's website ([here's Carbon's page](https://summerofcode.withgoogle.com/programs/2023/organizations/carbon-language)). Applications here won't be included.
